pages/consortium.html

```diff
@@ -0,0 +1,140 @@
+            <li>Advertisement of the member’s involvement in the Coq community.</li>
+        </ul>
+
+
+    </p>
+    <p>Partners declare a number of intensive users of Coq they represent, and the Consortium recommends a corresponding level of
+        service (a number of engineer hours, which can be used for support or training). The cost corresponds to that specific level of service. Academic members receive a 50% discount.</p>
+    <table>
+        <tr><td>Users</td><td>Engineer hours</td><td>1 year</td><td>2 years</td><td>3 years</td></tr>
+        <tr>
+            <td>
+                5-7
+            </td>
+            <td>
+                4h/user/year
+            </td>
+            <td>
+                € 2000/user
+            </td>
+            <td>
+                € 3600/user
+            </td>
+            <td>
+                € 4800/user
+            </td>
+        </tr>
+        <tr>
+            <td>
+                8-12
+            </td>
+            <td>
+                32h/year
+            </td>
+            <td>
+                € 16000
+            </td>
+            <td>
+                € 28800
+            </td>
+            <td>
+                € 38400
+            </td>
+        </tr>
+        <tr>
+            <td>
+                13-14
+            </td>
+            <td>
+                36h/year
+            </td>
+            <td>
+                € 18000
+            </td>
+            <td>
+                € 32400
+            </td>
+            <td>
+                € 43200
+            </td>
+        </tr>
+        <tr>
+            <td>
+                15+
+            </td>
+            <td>
+                40h/year
+            </td>
+            <td>
+                € 20000
+            </td>
+            <td>
+                € 36000
+            </td>
+            <td>
+                € 48000
+            </td>
+        </tr>
+    </table>
+
+    <p>Beside these services, the Consortium can establish contracts with members to audit code and emit
+a quality label if the audit is successful. The Consortium can also certify Coq related services like
+        training and support provided by members.</p>
+</div>
+</div>
+
+<div class="framework">
+<div class="frameworklabel">Hosted by a non-profit foundation</div>
+<div class="frameworkcontent">
+    <p>The Coq Consortium is hosted by a non-profit foundation associated to Inria providing engineering
+resources for the development of software created in research teams. This set-up makes it possible
+for the Coq Consortium to receive donations or establish specific support contracts.</p>
+</div>
+
+</div>
+
+<div class="framework">
+<div class="frameworklabel">How to join?</div>
+<div class="frameworkcontent">
+    <p>The Coq Consortium is currently being created by the Inria Foundation.
+        If you are interested, <please contact <a href="mailto:yves.bertot@inria.fr">Yves Bertot</a>
```

Comment:
Typo: this `<` is preventing the printing of "please contact".

Reply:
Thanks!